### ISSUE TITLE
[Anilist Tracker] An issue

### ISSUE THREAD

**Issue by everyonus** [z411/trackma#479]
For a while I was trying to figure out why not all of my anime was showing when I did ls and not being recognized  as part of my list as a whole. Trackma appears not to see anime entries that has been labeled has "Hide from status list" by the user.

**Comment by z411** [comment]
This is by AniList's design; that's what that function does, it removes the show from the status list. If you just want people to not see your item in the public list, use the "Private" function instead. Custom lists aren't implemented yet but I could work on it if you use them.

**Comment by everyonus** [comment]
I use custom lists with the "Hide from status list" function. Whereas the anime is still visible but not within the default (watching, planning, completed, paused, dropped) lists.

**Comment by Parent5446** [comment]
Just popping on to say I also use this feature, otherwise the show will show up twice, which can be annoying (once in my custom list and once in the normal status list). The only way to get around it would be to ensure _every_ show is in a custom list, and then to hide the normal "Watching" list from my homepage.

I suppose this cannot be fixed until the AniList API provides a way to fetch purely based on status, ignoring the "Hide from status lists" method.

**Comment by FichteFoll** [comment]
As a workaround, trackam could fetch all lists and deduplicate the entries, then categorize them by status. It's what I needed to do for anitogether.

https://github.com/FichteFoll/anitogether/commit/e4f14f9b16c4732f48ba1f7a4064f2c6a476532c
https://github.com/FichteFoll/anitogether/blob/abdb835b39c1031cce3f952aaa5c8116905358b6/src/App.vue#L135

**Comment by z411** [comment]
Indeed, another way would be to actually implement custom lists but that would entail duplicating entries and I'm not sure that would lead to good results right now. The shows are keyed by their ID in the list and we'd have to see how the tracker would handle this.

**Comment by uzuto** [comment]
Any progress on this??
